acceptance/telemetry/timeout/test.toml

```diff
@@ -6,4 +6,4 @@ Response.Body = '''
     "numProtoSuccess": 2
 }
 '''
-DelaySeconds = 5
+Delay = "5s"
```

Comment:
question - does it need to be changed? Can it still accept plain `5` or `0.01` ?

Reply:
No, it only accepts ints, and ints are nanos (the type of `time.Duration`).